Sources/StreamChat/Audio/AudioSessionConfiguring.swift

```diff
@@ -90,16 +92,18 @@ open class StreamAudioSessionConfigurator: AudioSessionConfiguring {
 
     /// Calling this method should activate the provided `AVAudioSession` for playback and record.
     ///
-    /// - Note: The method will check if the audioSession's category contains the `playAndRecord` capability
-    /// and if it doesn't it will activate it using the `.playbackAndRecord` category and `.default` for both mode
-    /// and policy.  OverrideOutputAudioPort is set to `.speaker`. The `record` capability is required
-    /// ensure that the output port can be set to `.speaker`.
+    /// - Note: This method uses the `.playAndRecord` category with `.default` mode and policy.
     open func activatePlaybackSession() throws {
         try audioSession.setCategory(
             .playAndRecord,
             mode: .default,
             policy: .default,
-            options: []
+            options: [
+                .defaultToSpeaker,
```

Comment:
The problem with the `.defaultToSpeaker` is that you cannot play audio from the earpiece which can be problematic(for reference, when setting the `.defaultToSpeaker` the earpiece isn't available at all, even from the control center output selection list).